blackbox/docs/admin/privileges.rst

```diff
@@ -5,16 +5,34 @@
 Privileges
 ==========
 
-The superuser is allowed to execute any statement without any privilege checks.
+To execute statements in CrateDB a user needs to have the required privileges
+to do so.
 
-The superuser uses ``GRANT``, ``DENY`` and ``REVOKE`` statements to control
-access to the resource. The privileges are either applied on the whole cluster
-or on instances of objects such as schema, or table.
+There is a superuser (``crate``) which has the privilege to do anything. The
+privileges of other users have to be managed using the ``GRANT``, ``DENY`` or
+``REVOKE`` statements.
 
-Currently, only ``DQL``, ``DML`` and ``DDL`` privileges can be granted.
+The privileges that can be granted, denied or revoked are:
+
+- ``DQL``
+- ``DML``
+- ``DDL``
+
+Skip to :ref:`privilege_types` for details.
+
+These privileges can be granted on different levels:
+
+- ``CLUSTER``
+- ``SCHEMA``
+- ``TABLE`` and ``VIEW``
+
+Skip to :ref:`hierarchical_privileges_inheritance` for details.
+
+
+A user with ``DDL`` on level ``CLUSTER`` can grant privileges they themselves
+have to other users as well. But once a grantor loses a privilege, all users
```

Comment:
Also not sure if we need this already. We could also extend this later on based on if there are any troubles with it.

Implementation is easier without this. (thinking minimal viable)